sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_client_manager.py

```diff
@@ -315,6 +324,8 @@ def __init__(
                 )
             )
             self._setup_failover_endpoints()
+            if load_balancing_enabled:
+                random.shuffle(self._replica_clients)
             return
```

Comment:
If we keep a common return statement, we can avoid duplicated logic in two blocks.

```
if connection_string and endpoint:
    clientWrapper = _ConfigurationClientWrapper.from_connection_string(...)
elif endpoint and credential:
    clientWrapper = _ConfigurationClientWrapper.from_credential(...)
else:
    raise ValueError("Please pass either endpoint and credential, or a connection string with a value.")

self._replica_clients.append(clientWrapper)
self._setup_failover_endpoints()
if load_balancing_enabled:
    random.shuffle(self._replica_clients)
return
```

Reply:
I think I wrote it the original way so the sync and async flows were at least a bit logically the same. But this works too, I've updated the sync version.